pkg/operator/resources/cluster/rbac.go

```diff
@@ -84,6 +84,21 @@ func getAdminPolicyRules() []rbacv1.PolicyRule {
 				"*",
 			},
 		},
+		{
+			APIGroups: []string{
+				"cdi.kubevirt.io",
+			},
+			Resources: []string{
+				"storageprofiles",
+			},
+			Verbs: []string{
+				"get",
+				"list",
+				"watch",
+				"patch",
+				"update",
```

Comment:
Don't see this tested anywhere

Reply:
Sorry, I meant `update` specifically.  Basically should storageprofile (and cdiconfig) be in the admin/edit/view clusterroles?

Reply:
That is good question, I also wonder if it makes sens. 
Correct me if I am wrong, but reading here [1], I see that default admin, edit and view roles are intended to be granted within particular namespace. 

The test [var _ = Describe("Aggregated role in-action tests", func() {](https://github.com/kubevirt/containerized-data-importer/blob/d0a58c266bcccc187f2f6aed1c30b584a3eda1f9/tests/rbac_test.go#L26) passes thanks to the following clusterRole and Binding https://github.com/kubevirt/containerized-data-importer/blob/d0a58c266bcccc187f2f6aed1c30b584a3eda1f9/pkg/operator/resources/cluster/rbac.go#L187.



https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings